library/pod-security-policy/seccompv2/template.yaml

```diff
@@ -0,0 +1,303 @@
+        source:
+          variables:
+          - name: containers
+            expression: 'has(variables.anyObject.spec.containers) ? variables.anyObject.spec.containers : []'
+          - name: initContainers
+            expression: 'has(variables.anyObject.spec.initContainers) ? variables.anyObject.spec.initContainers : []'
+          - name: ephemeralContainers
+            expression: 'has(variables.anyObject.spec.ephemeralContainers) ? variables.anyObject.spec.ephemeralContainers : []'
+          - name: allowAllProfiles
+            expression: |
+              has(variables.params.allowedProfiles) && variables.params.allowedProfiles.exists(profile, profile == "*")
+          - name: exemptImagePrefixes
+            expression: |
+              !has(variables.params.exemptImages) ? [] :
+                variables.params.exemptImages.filter(image, image.endsWith("*")).map(image, string(image).replace("*", ""))
+          - name: exemptImageExplicit
+            expression: |
+              !has(variables.params.exemptImages) ? [] : 
+                variables.params.exemptImages.filter(image, !image.endsWith("*"))
+          - name: exemptImages
+            expression: |
+              (variables.containers + variables.initContainers + variables.ephemeralContainers).filter(container,
+                container.image in variables.exemptImageExplicit ||
+                variables.exemptImagePrefixes.exists(exemption, string(container.image).startsWith(exemption)))
+          - name: unverifiedContainers
+            expression: |
+              (variables.containers + variables.initContainers + variables.ephemeralContainers).filter(container,
+                !variables.allowAllProfiles &&
+                !(container.image in variables.exemptImages))
+          - name: inputAllowedProfiles
+            expression: |
+              !has(variables.params.allowedProfiles) ? [] : variables.params.allowedProfiles
+          - name: derivedAllowedLocalhostFiles
+            expression: |
+              variables.inputAllowedProfiles.exists(profile, profile == "Localhost") ? variables.params.allowedLocalhostFiles.map(file, "Localhost/" + file) : []
+          - name: allowedProfiles
+            expression: |
+              variables.inputAllowedProfiles + variables.derivedAllowedLocalhostFiles
+          - name: allowAllLocalhostFiles
+            expression: |
+              variables.allowedProfiles.exists(profile, profile == "Localhost/*")
+          - name: hasPodSeccomp
+            expression: |
+              has(variables.anyObject.spec.securityContext) && has(variables.anyObject.spec.securityContext.seccompProfile)
+          - name: podLocalHostProfile
+            expression: |
+              variables.hasPodSeccomp && has(variables.anyObject.spec.securityContext.seccompProfile.localhostProfile) ? variables.anyObject.spec.securityContext.seccompProfile.localhostProfile : ""
+          - name: podSecurityContextProfile
```

Comment:
nit: `podSecurityContextProfileType`, that is what is actually being stored, and otherwise you are relying on users to read a plural (extra "s") to distinguish between two very different bits of data.